Class/Query.php

```diff
@@ -357,13 +372,31 @@ public function TestDetails()
             'result.result', 
             'result.score', 
             ]
-        )
-        
-        // ->where('examMaintain.examTitle')->is($title)
-       
-        ->all();
-        return $result;
+        );
+        // var_dump($query);die();
+        return $query->all();
     }
+    // public function TestDetails1()
```

Comment:
@Ratheesh392 Pls remove commented out codes throughout the PR

Reply:
Okay.